code/modules/atmospherics/environmental/LINDA_fire.dm

```diff
@@ -31,16 +31,13 @@
 		igniting = 1
```

Comment:
Filename for this still valid?

Reply:
I'll look at renaming/removing these once more of it gets moved into MILLA.